src/services/trainerMatchingService.js

```diff
@@ -0,0 +1,42 @@
+const { trainerMatchingDao } = require("../models");
+const { throwError } = require("../utils/throwError");
+
+const getTrainerProduct = async (userId, offset, limit) => {
+  //트레이너 확인 후 토큰 확인해서 구독 여부
+  const isTrainer = trainerMatchingDao.isTrainer(userId);
+  const isSubscript = trainerMatchingDao.isSubscript(userId);
+  const isSub = true;
+  if (isTrainer || isSubscript) {
+    isSub = false;
+  }
+  const data = await trainerMatchingDao.getTrainerMatching(offset, limit);
+  return { isSubscript: isSub, data: data };
+};
```

Comment:
```suggestion
const getTrainerProduct = async (userId, offset, limit) => {
  //트레이너 확인 후 토큰 확인해서 구독 여부
  const isTrainer = trainerMatchingDao.isTrainer(userId);
  const isSubscribed = trainerMatchingDao.isSubscribed(userId);
  const isSub = true;
  if (isTrainer || isSubscript) {
    isSub = false;
  }
  const data = await trainerMatchingDao.getTrainerMatching(offset, limit);
  return { isSubscribed: isSub, data: data };
};
```
- 구독 여부라면 subscribed 단어가 보다 적절해 보입니다.
- `if (isTrainer || isSubscript)` 조건이 다소 헷갈리는데, 트레이너이거나 subscription이 이미 있으면, isSub를 거짓으로 하는 로직이 맞나요?

Reply:
네네 맞습니다 subscribed로 바꿀게요 감사합니다!